packages/x-components/src/x-installer/x-installer/types.ts

```diff
@@ -79,3 +79,19 @@ export interface ExtraPluginsOptions {
  * @public
  */
 export type VueConstructorPartialArgument = Partial<ConstructorParameters<VueConstructor>[0]>;
+
+/**
+ * Interface for the returned type of the {@link XInstaller.init} function.
+ *
+ * @public
+ */
+export interface InitWrapper {
+  /** The root Vue instance of the application. */
+  app: Vue;
+  /** The {@link XAPI} to expose globally. */
+  api: XAPI;
+  /** The {@link XBus} used in the {@link XPlugin}. */
```

Comment:
Remember that this two properties were optional before, and thanks for extracting them.

```suggestion
  app?: Vue;
  /** The {@link XAPI} to expose globally. */
  api?: XAPI;
  /** The {@link XBus} used in the {@link XPlugin}. */
```